tutorials/04-hidden-markov-model/04_hidden-markov-model.jmd

```diff
@@ -100,8 +96,8 @@ Time to run our sampler.
 
 
 ```julia
-g = Gibbs(HMC(0.001, 7, :m, :T), PG(20, :s))
-c = sample(BayesHmm(y, 3), g, 100);
+g = Gibbs(HMC(0.01, 50, :m, :T), PG(120, :s))
+chn = sample(BayesHmm(y, 3), g, 1000);
```

Comment:
Mostly curious, what's the run time on this one?

Reply:
10min on my machine. Is that within bounds? I tried a few other parameter settings and it was quite hard to get it to converge for lower values to be honest. Otherwise, I would just try and reduce the chain length.

Reply:
Yeah, I think 10 is perfectly fine.